Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Xamarin.Forms.Controls.Issues.Shared.projitems

```diff
@@ -19,6 +19,9 @@
     <Compile Include="$(MSBuildThisFileDirectory)CollectionViewHeaderFooterTemplate.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)CollectionViewHeaderFooterView.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)CollectionViewItemsUpdatingScrollMode.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Issue5108.xaml.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Issue5500.cs" />
```

Comment:
```
Build Windows Phase debug: "C:\Program Files\dotnet\sdk\2.2.109\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.DefaultItems.targets(247,5): Error NETSDK1022: Duplicate 'Compile' items were included. The .NET SDK includes 'Compile' items from your project directory by default. You can either remove these items from your project file, or set the 'EnableDefaultCompileItems' property to 'false' if you want to explicitly include them in your project file. For more information, see https://aka.ms/sdkimplicititems. The duplicate items were: 'D:\a\1\s\Xamarin.Forms.Controls.Issues\Xamarin.Forms.Controls.Issues.Shared\Issue5500.cs'; 'D:\a\1\s\Xamarin.Forms.Controls.Issues\Xamarin.Forms.Controls.Issues.Shared\Issue8008.cs'"
```

Reply:
Hi @samhouts . I'm sorry but I don't understand what is going on here... Could you please clarify?
Isn't it weird that files 'Issue5500.cs' and 'Issue8008.cs' are added here as, in fact all I did with my origional PR was adding Issue5108.(xaml).cs (and fixing the issue).
Also, about the above build error, should I just remove these lines from projitems?
Sorry about my lag of understanding this. :-)

Reply:
The error is a bit vague, but it looks like a merge gone wrong or something. If I look at the [commits tab](https://github.com/xamarin/Xamarin.Forms/pull/7518/commits) I see some from Shane in there. I guess that's not right?